bgpd/bgp_main.c

```diff
@@ -469,6 +476,23 @@ main (int argc, char **argv)
 	  if (vty_port <= 0 || vty_port > 0xffff)
 	    vty_port = BGP_VTY_PORT;
 	  break;
+	case OPTION_VTYSOCK:
+	  vty_sock_name = strrchr (BGP_VTYSH_PATH, '/');
```

Comment:
Anytime we have cut-n-paste code with a variable name being changed, it is proper to make it a function call.  So I would prefer if we created a function call